src/Avalonia.Visuals/Media/FontFamily.cs

```diff
@@ -112,7 +114,13 @@ public static FontFamily Parse(string s)
 
                 case 2:
                     {
-                        return new FontFamily(segments[1], new Uri(segments[0], UriKind.RelativeOrAbsolute));
+                        var uri = segments[0].StartsWith("/")
+                                      ? new Uri(segments[0], UriKind.Relative)
+                                      : new Uri(segments[0], UriKind.RelativeOrAbsolute);
+
+                        return uri.IsAbsoluteUri
+                                   ? new FontFamily(segments[1], uri)
+                                   : new FontFamily(segments[1], uri, baseUri);
```

Comment:
It should be safe to always pass baseUri, since AssetLoader detects uri kind internally.

Reply:
I am not sure if that is the case. I do not want to store the base uri when the path is absolute.